src/app/containers/Article/index.jsx

```diff
@@ -58,12 +62,14 @@ ArticleContainer.propTypes = {
   loading: bool,
   error: string,
   data: shape(articlePropTypes),
+  bbcOrigin: string,
```

Comment:
It seems that the `bbcOrigin` should be a required param as it has a default value `getOriginContext.js`

Reply:
The required-ness of a prop shouldn't be influenced by what use it, or pass to it.

Reply:
Fair point 👍